app/models/concerns/hyrax/solr_document_behavior.rb

```diff
@@ -68,7 +68,8 @@ def collection?
 
     # Method to return the ActiveFedora model
     def hydra_model
-      first(Solrizer.solr_name('has_model', :symbol)).constantize
+      value = first(Solrizer.solr_name('has_model', :symbol))
+      return value.constantize if value
```

Comment:
I don't know that `safe_constantize` is going to be useful in this case because we can't call `nil.safe_constantize`.   I think it's a good idea to raise a NameError if the model field actually can't be constantized.